src/native/corehost/corehost.proj

```diff
@@ -74,9 +74,13 @@
       <CMakeBuildDir>$(IntermediateOutputRootPath)corehost\cmake\</CMakeBuildDir>
       <BuildScript>$([MSBuild]::NormalizePath('$(MSBuildThisFileDirectory)', 'build.sh'))</BuildScript>
 
+      <CommitHash Condition="('$(CommitHash)' == '') and ('$(DotNetGitCommitHash)' != '')">$(DotNetGitCommitHash)</CommitHash>
+      <CommitHash Condition="('$(CommitHash)' == '') and ('$(SourcerevisionId)' != '')">$(SourceRevisionId)</CommitHash>
```

Comment:
nit: I find it easier to read when the property just gets set based on being empty and the order indicates the precedence.
```suggestion
      <CommitHash Condition="'$(CommitHash)' == ''">$(DotNetGitCommitHash)</CommitHash>
      <CommitHash Condition="'$(CommitHash)' == ''">$(SourceRevisionId)</CommitHash>
```